.travis.yml

```diff
@@ -6,23 +6,23 @@ branches:
   only: master
 
 rvm:
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
+  - 2.4
+  - 2.5
+  - 2.6
 
 gemfile:
- - gemfiles/rails3.2.gemfile
- - gemfiles/rails4.1.gemfile
- - gemfiles/rails4.2.gemfile
```

Comment:
we still want to run these, right?

Reply:
I think we want 4.2 but nothing below that, do we?